connect/cli/plugins/project/report/templates/add/package_dir/report_dir/templates/pdf/template.html.j2.j2

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <link href="reports/my_awesome_report_pdf/templates/pdf/template.css" rel="stylesheet">
+        <title>Report</title>
+        <meta name="description" content="Requests Executive Report">
```

Comment:
It should be <meta name="description" content="{{ initial_report_description}}">
Probably we also need some jinja filter to ensure that the initial report description as no newline otherwise the generated html will be invalid.